### PR TITLE
Django-1.7+

### PR DESCRIPTION
**Pull request by Kasperfred**

### CODE REVIEW

File: imagefit/models.py [hunk 1]
```diff
@@ -23,7 +25,7 @@ def __init__(self, path, cache=None, cached_name=None, *args, **kwargs):
         self.cached_name = cached_name
 
         # force RGB
-        if self.pil.mode not in ('L', 'RGB', 'LA', 'RGBA'):
+        if self.pil.mode not in ('L', 'RGB'):
```
Comment (vinyll) [inline imagefit/models.py]: Why removing alpha transparency?

File: imagefit/models.py [hunk 2]
```diff
@@ -1,12 +1,14 @@
 from __future__ import division
-from imagefit.conf import ext_to_format, settings
+from imagefit.conf import settings
 from PIL import Image as PilImage
 
 import mimetypes
+
 try:
-    import StringIO
+    from StringIO import StringIO
```
Comment (vinyll) [inline imagefit/models.py]: You use `BytesIO` further down but it won't be available if `StringIO` does not fail importing.

File: imagefit/models.py [hunk 3]
```diff
@@ -72,8 +74,9 @@ def render(self):
         if self.is_cached:
             return self.cache.get(self.cached_name)
         else:
-            image_str = StringIO.StringIO()
-            self.pil.save(image_str, ext_to_format(self.cached_name))
+            image_str = BytesIO()
+            # not much other supports than png, yet works
+            self.pil.save(image_str, 'png')
```
Comment (vinyll) [inline imagefit/models.py]: Why removing the format detection?

File: imagefit/views.py [hunk 4]
```diff
@@ -57,11 +52,12 @@ def resize(request, path_name, format, url):
             " \"%s\" is neither a \"WIDTHxHEIGHT\" format nor a key in " +
             "IMAGEFIT_PRESETS." % format
         )
-
+    
     # Resize and cache image
     if preset.get('crop'):
         image.crop(preset.get('width'), preset.get('height'))
     else:
+        print ("PRINTING", preset.get('width'), preset.get('height'))
```
Comment (vinyll) [inline imagefit/views.py]: Remove or use `logger`

File: README.md [hunk 5]
```diff
@@ -6,7 +6,7 @@ Imagefit allows you to render an image in a template and specify its dimensions.
 It preserves the original image file.
 
 It is compatible with various sources of images such as django-filebrowser's
-FileBrowseField, user uploaded images, static images, …
+FileBrowseField, user uploaded images, static images, ...
```
Comment (vinyll) [inline README.md]: I'd rather have an ellipsis character than 3 dots to emulate it. Did you meet any trouble with that?

File: imagefit/conf.py [hunk 6]
```diff
@@ -1,47 +1,41 @@
 from django.conf import settings
-from appconf import AppConf
+from django.conf import LazySettings
 
 import tempfile
 import os
 
 
-class ImagefitConf(AppConf):
+class Settings(LazySettings):
```
Comment (vinyll) [inline imagefit/conf.py]: I guess this new approach is cleaner :)

**Comment by vinyll** [comment]
Conflict: I guess it would need a _rebase_ over _master_.

**Comment by vinyll** [comment]
Do you still want to work on this PR @Kasperfred or should I take over / close?

**Comment by Kasperfred** [comment]
Oh, sorry. I think a better solution is already implemented @vinyll I'll close this request.

**Comment by vinyll** [comment]
Could you share with us what is that solution with a better implementation @Kasperfred?

**Comment by Kasperfred** [comment]
Uh I was referring to the master branch @vinyll. This request is the solution I use in my own project which works fine for what I need. 

Do you have a timeframe for when you will make the new version available through pip?

**Comment by vinyll** [comment]
release 0.5 is now on pypi @Kasperfred.

**Pull request closed without merging**